www/js/metrics.js

```diff
@@ -1160,11 +1160,25 @@ angular.module('emission.main.metrics',['nvd3', 'emission.services', 'ionic-date
     getData();
   }, 1)
 
+  $scope.sendStats = function() {
```

Comment:
why do we need a `sendStats` and why do we record `OPENED_APP` in it?

Reply:
Forgot to remove this